db/migrate/20160627104436_use_https_name_change_in_site_settings.rb

```diff
@@ -0,0 +1,13 @@
+class UseHttpsNameChangeInSiteSettings < ActiveRecord::Migration
+  def up
+    SiteSetting.transaction do
+      old_setting = SiteSetting.find_by(name: 'use_https')
```

Comment:
nahh 

you don't need the transaction there and do the whole change using raw sql, we get bitten long term with these kind of migrations